foreman/data_refinery_foreman/surveyor/test_harmony.py

```diff
@@ -270,8 +265,8 @@ def test_sra_lots(self):
         for accession in lots:
             try:
                 metadata = SraSurveyor.gather_all_metadata(accession)
-                harmonized = harmonize([metadata])
-            except UnsupportedDataTypeError as udte:
+                harmonize([metadata])
```

Comment:
On the other hand - not sure what this pattern is trying to do. Run harmonize but then we're ok with UnsupportedDataTypeError? Looks like not a great test pattern. Seems better to test one where we expect success and another where we assert that this error is raised.

Reply:
I agree, I think we should add this on a different PR and leave this one only for unused variables.